composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dektrium/yii2-user",
+    "name": "dabilite/yii2-user",
     "description": "Flexible user registration and authentication module for Yii2",
```

Comment:
Disagree. Merging back into dektrium (hopefully soon) will be as easy as replacing this single string again.